Document-Processing/Excel/Spreadsheet/Javascript-ES6/comment.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: Comment in EJ2 TypeScript Spreadsheet control | Syncfusion
+description: Learn here all about Comment feature in Syncfusion EJ2 TypeScript Spreadsheet control of Syncfusion Essential JS 2 and more.
+platform: document-processing
+control: Comment
+documentation: ug
+---
+
+# Comment in EJ2 TypeScript Spreadsheet control
+
+The **Comment** feature allows you to add remarks and **replies** to cells, making it easier to provide feedback or enabling contextual discussions without altering cell values. Comment are distinct from **Notes** and includes advanced capabilities such as replies, resolve/reopen, and an optional **Comments Review Pane** to maintain clear communication and streamline review workflows. When a cell contains a comment, a small comment indicator appears in its corner. Hover over the indicator to preview the comment.
```

Comment:
The **Comments** feature allows you to add feedback to cells without changing their values, supporting threaded **replies** for contextual discussions. Unlike [Notes](), Comments include advanced review tools such as **resolve** and **reopen** to track status, plus an optional **Comments Review Pane** for browsing and managing threads.

Cells with comments display a small corner indicator; hover to preview the comment or select the cell to view details and reply. This provides a clear, collaborative workflow while keeping data intact.